code/modules/mob/dead/observer/logout.dm

```diff
@@ -1,4 +1,5 @@
 /mob/dead/observer/Logout()
+	client.images -= ghostimages

```

Comment:
If the person is logging out the client goes away anyways, so for what porpoise?